main.go

```diff
@@ -502,20 +519,10 @@ func makeInitialRequest(out output, transport transport.Transport, serializer en
 	}
 
 	// Print the initial output body.
-	outSerialized := map[string]interface{}{
-		"body": responseMap,
-	}
-	if len(response.Headers) > 0 {
-		outSerialized["headers"] = response.Headers
-	}
+	outSerialized["body"] = responseMap
```

Comment:
The comment `// Print the initial output body.` is not up do date anymore, please move it to the defer function